testing/gcin/APKBUILD

```diff
@@ -0,0 +1,64 @@
+# Contributor: Felix Yan <felixonmars@archlinux.org>
+# Contributor: Gaetan Bisson <bisson@archlinux.org>
+# Contributor: Andreas Radke <andyrtr@archlinux.org>
+# Contributor: damir <damir@archlinux.org>
+# above are from Arch Linux
+# Contributor: Daniel YC Lin <dlin.tw@gmail.com>
+# Maintainer: Daniel YC Lin <dlin.tw@gmail.com>
+
+pkgname=gcin
+pkgver=2.8.4
+pkgrel=0
+pkgdesc='Input method server supporting various input methods'
+url='http://hyperrate.com/dir.php?eid=67'
+license="LGPL"
+arch="all"
+#depends="gtk2 libxtst"
```

Comment:
What’s the purpose of such “comment”? Please remove dead/irrelevan code instead of commenting it out.